app/src/components/market/common/list_item/index.tsx

```diff
@@ -129,12 +134,39 @@ export const ListItem: React.FC<Props> = (props: Props) => {
   const percentages = calcPrice(outcomeTokenAmounts)
   const indexMax = percentages.indexOf(Math.max(...percentages))
 
+  let realitioScalarAdapter
+  if (context.networkId === 1) {
+    realitioScalarAdapter = REALITIO_SCALAR_ADAPTER_ADDRESS.toLowerCase()
```

Comment:
The addresses are actually in `util/networks` already, and afaik `realitioTimeout` is the same, but I've made a note to make sure the reality data is accurate

Reply:
Oh yeah they are too! Just to clarify my comment in case it was confusing - At some point we should add something like: `getRealitioScalarAdapter(context.networkId)` so we don't have to update if statements as we add new networks 👍